### PR TITLE
Leanback project

### PR DESCRIPTION
**Pull request by bspinner**

### CODE REVIEW

File: MvvmCross.Droid.Support.V17.Leanback/MvvmCross.Droid.Support.V17.Leanback.csproj [hunk 1]
```diff
@@ -1,127 +1,128 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (bspinner) [inline MvvmCross.Droid.Support.V17.Leanback/MvvmCross.Droid.Support.V17.Leanback.csproj:1]: Whyever the whole file is considered new.

**Comment by martijn00** [comment]
The whole Cirrious name is dropped since my last PR's. Could you keep your files in sync with that?

**Comment by bspinner** [comment]
Should be clean now?

**Comment by martijn00** [comment]
Seems correct! Thanks!

**Pull request merged**